ramls/item.json

```diff
@@ -128,6 +132,34 @@
             "type": "string",
             "description": "Text to display"
           },
+          "source": {
+            "type": "object",
+            "description": "The user who added the note",
```

Comment:
Is this always the user who added the note, or does it change when the note is updated (to the user who last updated the note)?